package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "puppeteer-utils",
+  "version": "0.0.5",
+  "description": "Puppeteer utils and config for WP",
+  "private": true,
```

Comment:
I think we can get rid of this line since the repo is public.

Reply:
done